pilot/lib/pilot/build_manager.rb

```diff
@@ -368,12 +368,17 @@ def should_update_localized_build_information?(options)
     end
 
     def reject_build_waiting_for_review(build)
-      waiting_for_review_build = build.app.get_builds(filter: { "betaAppReviewSubmission.betaReviewState" => "WAITING_FOR_REVIEW" }, includes: "betaAppReviewSubmission,preReleaseVersion").first
+      waiting_for_review_build = build.app.get_builds(
+        filter: { "betaAppReviewSubmission.betaReviewState" => "WAITING_FOR_REVIEW,IN_REVIEW",
```

Comment:
Nitpick: Not sure if the scope is the same or if the visibility between `spaceship` and `pilot` would allow it, but if it is, would you consider DRYing these "magic strings" in favor of https://github.com/fastlane/fastlane/blob/df12128496a9a0ad349f8cf8efe6f9288612f2cb/spaceship/lib/spaceship/connect_api/models/app_info.rb#L44-L55

Reply:
> Not sure if the scope is the same

After looking at the `connect_api/models` folder, I don't think the scope is the same. It's likely that the words used by the API would not change, but it would not be appropriate to use one for the other.

Reply:
> It's likely that the words used by the API would not change

@mokagio You've clearly never worked with Apple's ASC API 😅😁 (they've changed the API without warning—nor updating their doc—at multiple occasions in the past sadly, requiring us to update _fastlane_ in a hurry sometimes to account for the sudden change 😑)

Reply:
Can we keep the DRY refactors for another PR? It slows down the adoption of fixes.
We can limit the scope of this PR to the actual fix.
Would love to see this change in, it's been a problem for ages.